navidrome/egg-navidrome.json

```diff
@@ -0,0 +1,142 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-12-27T22:06:30+01:00",
+    "name": "Navidrome",
+    "author": "felixseiboldt@gmail.com",
+    "description": "Navidrome Music Server - A lightweight music server for your personal collection.",
+    "features": null,
+    "docker_images": {
+        "devilapi\/navidrome-pterodactyl:latest": "devilapi\/navidrome-pterodactyl:latest"
```

Comment:
This is not allowed. The image must be owned by pelican. So please pr it if it meeds the requirements